crates/ruff/src/rules/mccabe/rules.rs

```diff
@@ -50,13 +50,18 @@ use ruff_python_ast::source_code::Locator;
 pub struct ComplexStructure {
     pub name: String,
     pub complexity: usize,
+    pub max_complexity: usize,
 }
 
 impl Violation for ComplexStructure {
     #[derive_message_formats]
     fn message(&self) -> String {
-        let ComplexStructure { name, complexity } = self;
-        format!("`{name}` is too complex ({complexity})")
+        let ComplexStructure {
+            name,
+            complexity,
+            max_complexity,
+        } = self;
+        format!("`{name}` is too complex ({complexity} > {max_complexity})")
```

Comment:
Do you mind changing those Pylint messages too, like `too_many_branches`, to use this style?

Reply:
Sure 👍

Reply:
Done (e17a361) and updated the title too